selfdrive/loggerd/SConscript

```diff
@@ -1,7 +1,9 @@
 Import('env', 'arch', 'cereal', 'messaging', 'common', 'visionipc', 'gpucommon')
 
-src = ['loggerd.cc', 'logger.cc']
-libs = ['zmq', 'capnp', 'kj', 'z',
+
+logger_lib = env.Library('logger', ["logger.cc"])
+src = ['loggerd.cc']
```

Comment:
just put this directly in the env.Program call

Reply:
no,we can't

Reply:
why not?

Reply:
`src` is modified by the following conditional statement

Reply:
ah, didn't see it in the diff view :)